session/session.go

```diff
@@ -145,6 +146,18 @@ type StmtHistory struct {
 	history []*stmtRecord
 }
 
+func init() {
+	executor.CreateSessionForBRIEFunc = func(sctx sessionctx.Context) (checkpoints.Session, error) {
```

Comment:
Add a comment for this

Reply:
`CreateSessionForBRIEFunc` has comment at its declared place. Should I move here or keep both?

Reply:
ok, no need to addess this comment